.eslintrc.json

```
@@ -9,8 +9,14 @@
         "prettier"
     ],
     "parser": "@typescript-eslint/parser",
+    "ignorePatterns": "*.config.js",
     "parserOptions": {
-        "project": "tsconfig.json",
+        "projects": [
+            "src/tsconfig.json",
+            "tests/tsconfig.json",
+            "stories/tsconfig.json",
```

Let's make this as clear as possible, since this it took me longer than I thought it would reviewing the rollup config to know what you were building there
```suggestion
            "src/tsconfig.lib.json",
            "tests/tsconfig..spec.json",
            "stories/tsconfig.stories.json",
```

---

then we can name tsconfig.base => tsconfig.json

---

although i think currently if you keep them all as `tsconfig.json` then vscode will correctly typecheck each file with the relevant tsconfig right ?

---

we can't name `tsconfig.base.json` -> `tsconfig.json`, because there's already a `tsconfig.json` at the root which aggregates all the other `tsconfig.json` files which is the one used by vscode for intellisense and etc.

Regarding the names, it makes no difference actually, `tsconfig.json` on the root is the one being used by vscode if I'm not mistaken

---

Oh OK I got it, let's rename the specific tsconfigs more appropriately